### ISSUE TITLE
Instructions on changing Github Project Boards visibility outdated?

### ISSUE THREAD

**Issue by obtusedev** [github/docs#9493]
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

[Change Project Board Visibility](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/managing-project-boards/changing-project-board-visibility)

[Direct link to markdown](https://github.com/github/docs/blob/main/content/issues/organizing-your-work-with-project-boards/managing-project-boards/changing-project-board-visibility.md)

### What part(s) of the article would you like to see updated?

Changing project board visibility:

> Navigate to the project board you want to make public or private.
> On the top-right side of the project board, click Menu.
Click ... then click Settings. 

The settings option in the menu is not avaliable to me.  
I don't know if something has changed since the last time this article was updated.

### Additional information

Below I have attached a photo of my menu.
![Screenshot](https://user-images.githubusercontent.com/66740598/131767252-4e644a87-ae05-44ff-bc56-db70f65b5868.png)

As you can see there is no settings option to change the Project Board's visibility.
Now compared that with the image on the documentation.
![Screenshot](https://docs.github.com/assets/images/help/projects/project-board-settings.png)

**Comment by ramyaparimi** [comment]
@obtusedev 
Thanks so much for opening an issue! I'll triage this for the team to take a look :eyes:

**Comment by lecoursen** [comment]
Hi @obtusedev! I do see Settings in my test project:

![image](https://user-images.githubusercontent.com/14935376/131900211-97cf7274-5069-48a9-8191-a54a6846c6ee.png)

Do you have admin permissions to the project board you're trying to change? You do need those permissions to change the visibility. If you have them, you can reach out to support at https://support.github.com to find out what else might be going wrong. Thanks for your interest in contributing to the docs!

**Comment by obtusedev** [comment]
> Do you have admin permissions to the project board you're trying to change? You do need those permissions to change the visibility.

I created the repo and board so I'm assuming I have admin permissions. Is that an incorrect assumption?  
Where would I go to check for if I have admin permissions? I looked around the repo settings but can't find it.

I took a look at the docs and it talks about going to the `Your organizations` tab by clicking the profile pic but I also don't see this setting. This might be because I don't recall joining any organizations.

Thanks for your time.
